custom-elements/reactions/HTMLBaseElement.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<head>
+<title>Custom Elements: CEReactions on HTMLBaseElement interface</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<meta name="assert" content="href, target of HTMLBaseElement interface must have CEReactions">
+<meta name="help" content="https://html.spec.whatwg.org/#the-base-element">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/custom-elements-helpers.js"></script>
+</head>
+<script>
+
+testReflectAttribute('href', 'href', '/', '/', 'http://example.com/', 'http://example.com/', 'href on HTMLBaseElement', 'base', HTMLBaseElement);
+testReflectAttribute('target', 'target', '_blank', '_blank', '_self', '_self', 'target on HTMLBaseElement', 'base', HTMLBaseElement);
+
+function testReflectAttribute(jsAtName, coAtName, jsAtValue1, coAtValue1, jsAtValue2, coAtValue2, name, elementName, interfaceName) {
```

Comment:
This function should probably be added to custom-elements-helpers.js instead.

Reply:
@rniwa, many thanks for your review, I think [reactions.js](https://github.com/web-platform-tests/wpt/blob/master/custom-elements/reactions/resources/reactions.js) is more appropriate, and I've addressed it in latest commit and replaced the `testReflectAttrWithParentNode` function in [HTMLButtonElement.html](https://github.com/web-platform-tests/wpt/blob/master/custom-elements/reactions/HTMLButtonElement.html) as well, PTAL.